docs/sources/reference/api/docker_remote_api_v1.19.md

```diff
@@ -974,17 +972,17 @@ Status Codes:
     When using the TTY setting is enabled in
     [`POST /containers/create`
     ](/reference/api/docker_remote_api_v1.9/#create-a-container "POST /containers/create"),
-    the stream is the raw data from the process PTY and client's stdin.
+    the stream is the raw data from the process PTY and client's `stdin`.
     When the TTY is disabled, then the stream is multiplexed to separate
-    stdout and stderr.
+    `stdout` and `stderr`.
 
     The format is a **Header** and a **Payload** (frame).
 
     **HEADER**
 
-    The header will contain the information on which stream write the
-    stream (stdout or stderr). It also contain the size of the
-    associated frame encoded on the last 4 bytes (uint32).
+    The header contains the information which the stream writes (`stdout` or
+    `stderr`). It also contain the size of the associated frame encoded on the
+    last 4 bytes (`uint32`).
 
     It is encoded on the first 8 bytes like this:
```

Comment:
Hmm, I think 8 bytes is part of a measurement.
![image](https://cloud.githubusercontent.com/assets/1347209/7871412/8a43cf8c-0544-11e5-907e-6ff5295f7248.png)